### ISSUE TITLE
Recurring log entries: Camel IDEA plugin cannot download camel-catalog with version 2.17.0.custom. Will fallback and use version 2.19.0-SNAPSHOT

### ISSUE THREAD

**Issue by rolandio** [camel-idea-plugin/camel-idea-plugin#307]
In our project we use the camel version from the server directly. We have tons of modules and using this plugin I get also tons of recurring log entries regarding my unknown version and that the camel-catalog from 2.19.0-SNAPSHOT is used instead.

Please download the catalog **only once**. And if downloaded, keep it until a version change is made, instead of downloading it again and again. I am not sure what triggers the log entries (maybe the maven build?), but I already got 500 additional entries just while I was writing this issue. (Additionally, but maybe worth its own issue: what about having plugin properties where one can map custom camel versions to actual versions?)

intellij idea: 2017.2 EAP
camel idea plugin: 4.0
camel: 2.17.0.REDHAT-630187 (Fuse 6.3)

**Comment by davsclaus** [comment]
You can either

- turn download off in the preference.
- add the 3rd party maven repo to the jboss fuse repo in the preference so it can find the Camel Fuse JARs
- or add the 3rd part maven repo to your pom.xml file of your project

**Comment by rolandio** [comment]
Thanks for your reply. I tried your first suggestion (turning off the download via 'Allow downloading camel-catalog over the internet'). But instead of the warnings I now have lots of `INFO`-log messages containing `Camel IDEA plugin is using camel-catalog version 2.19.0`. So it isn't that much better.
Again, if I reimport the maven projects the messages are repeated, even though nothing changed. Again, I now have like 500 additional info entries (just be reimporting some modules). Will try the variant with the fuse repo later. For now the log messages are what annoys me the most.

**Comment by davsclaus** [comment]
Paste a screenshot. 

And can you tell about the project you load in IDEA, does it have a lot of source code / maven modules or what is it ?

**Comment by davsclaus** [comment]
You can turn off that ballon popup in IDEA preference
![screen shot 2017-05-23 at 10 46 35 am](https://cloud.githubusercontent.com/assets/477100/26346007/24a696c6-3fa5-11e7-8023-3e7755c35b08.png)

**Comment by rolandio** [comment]
![After reimporting](https://cloud.githubusercontent.com/assets/12057039/26346266/f1000554-3fa5-11e7-89d4-a6e5213009c8.png)
Yes it has lots of modules (like 50-100 parent modules and 5-6 submodules each). Now counting 900 log entries ;-)

**Comment by davsclaus** [comment]
I did a new 0.4.1 release which hopefully wont notify so many time. You can give that a try when you its available from IDEA plugin updater

**Comment by rolandio** [comment]
Already updated. Thanks!
Works much better now 👍
